tumblr2hugomarkdown.py

```diff
@@ -69,7 +69,7 @@ def processPostBodyForImages(postBody, imagesPath, imagesUrlPath):
 		if os.path.exists(concreteImagePath):
 			# This image was already downloaded, so just replace the URL in body
 			postBody = postBody.replace(concreteImageUrl, imageOutputUrlPath)
-			print "Found image url", concreteImageUrl, "already downloaded to path", concreteImagePath
+			print ("Found image url", concreteImageUrl, "already downloaded to path", concreteImagePath)
 		else:
 			# Download the image and then replace the URL in body
 			imageContent = urllib2.urlopen(concreteImageUrl).read()
```

Comment:
I think you forgot to update this?

```suggestion
			imageContent = requests.get(concreteImageUrl).content
```